examples/tests/valid/tfaws_reference_bucket.main.w

```diff
@@ -0,0 +1,12 @@
+bring tf_aws;
+bring cloud;
+bring util;
+
+
+if util.env("WING_TARGET") == "tf-aws" {
+  let bucket = tf_aws.Bucket.fromBucketName(this, "some-existing-bucket-name");
+
+  new cloud.Function(inflight (msg: str) => {
+    bucket.put("somefile.txt", "wow this goes in an existing bucket!");
+  });
+}
```

Comment:
The underscore looks a bit out of place to me since we've been sticking to camelCase for most Wing example code. @eladb thoughts? Who's the owner of Wing code style?
```suggestion
bring tfaws;
bring cloud;
bring util;


if util.env("WING_TARGET") == "tf-aws" {
  let bucket = tfaws.Bucket.fromBucketName(this, "some-existing-bucket-name");

  new cloud.Function(inflight (msg: str) => {
    bucket.put("somefile.txt", "wow this goes in an existing bucket!");
  });
}
```

Reply:
Yes, this should be `tfaws` I think.